zhaquirks/tuya/builder/__init__.py

```diff
@@ -65,6 +66,10 @@ class TuyaCO2Concentration(CarbonDioxideConcentration, TuyaLocalCluster):
     """Tuya Carbon Dioxide concentration measurement."""
 
 
+class TuyaElectricalConductivity(ElectricalConductivity, TuyaLocalCluster):
```

Comment:
This was taken from https://github.com/zigpy/zha-device-handlers/issues/2565#issuecomment-1712541101